src/icons/icons.stories.html

```diff
@@ -7,6 +7,59 @@ <h2>Icons</h2>
         <use xlink:href="#omrs-icon-visibility"></use>
       </svg>
     </div>
+    <div>
```

Comment:
Since we're going to have a lot of svgs, it might be nice to pass in the names as an array to this html template, and then not have to modify this story every time we add a new icon.

icons.stories.js:
```js
storiesOf("OpenMRS Styleguide", module).add("Icons", () => {
  const icons = {
    Visibility: "visibility"
    Menu: "menu",
    Search: "search",
    "Arrow back": "arrow-back",
    "Arrow downward": "arrow-downward",
    "Arrow upward": "arrow-upward",
    "Chevron left": "chevron-left",
    "Chevron right": "chevron-right",
    "Supervised user circle": "supervised-user-circle"
  }

  const iconNames = Object.values(icons)

  const iconChoice = select(
    "Icon",
    icons,
    "visibility"
  );
  const href = `#omrs-icon-${iconChoice}`;
  return htmlStory(ejs.render(html, { href, iconNames }));
});
```

icons.stories.html:
```ejs
<% for (var i = 0; i < iconNames.length; i++) {  %>
  <div>
    <div>omrs-icon-<%- iconNames[i] %></div>
      <svg class="omrs-icon" role="img">
        <use xlink:href="#omrs-icon-<%- iconNames[i] %>"></use>
      </svg>
    </div>
<% } %>
```

Reply:
Done